packages/sdk/src/conversationWithCloudAdapter/sso/botSsoExecutionDialog.browser.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { formatString } from "../../util/utils";
+import { ErrorWithCode, ErrorCode, ErrorMessage } from "../../core/errors";
+import { TeamsFx } from "../../core/teamsfx.browser";
+import { BotSsoExecutionDialogHandler, TriggerPatterns } from "../interface";
+import { StatePropertyAccessor, TurnContext, Storage } from "botbuilder";
+import { OnBehalfOfCredentialAuthConfig } from "../../models/configuration";
+import { TeamsBotSsoPromptSettings } from "../../index.browser";
+/*
+ * Sso execution dialog, use to handle sso command
+ */
+export class BotSsoExecutionDialog {
+  /**
+   * Constructor of BotSsoExecutionDialog
+   *
+   * @remarks
+   * Can Only works in in server side.
+   */
+  constructor(
+    dedupStorage: Storage,
+    ssoPromptSettings: TeamsBotSsoPromptSettings,
+    teamsfx: TeamsFx,
+    dialogName?: string
+  );
+  constructor(
+    dedupStorage: Storage,
+    ssoPromptSettings: TeamsBotSsoPromptSettings,
+    authConfig: OnBehalfOfCredentialAuthConfig,
+    initiateLoginEndpoint: string,
+    dialogName?: string
+  );
+  constructor(
+    dedupStorage: Storage,
+    ssoPromptSettings: TeamsBotSsoPromptSettings,
+    authConfig: TeamsFx | OnBehalfOfCredentialAuthConfig,
+    ...args: any
+  ) {
+    throw new ErrorWithCode(
+      formatString(ErrorMessage.BrowserRuntimeNotSupported, "BotSsoExecutionDialog"),
+      ErrorCode.RuntimeNotSupported
+    );
+  }
+
+  // eslint-disable-next-line no-secrets/no-secrets
```

Comment:
Are these eslint-disable-* comments necessary? It looks like there's no secret next line.

Reply:
![image](https://user-images.githubusercontent.com/45935381/209285866-02f7003d-8582-4e98-bef6-817ef0aa9761.png)
Seems `BotSsoExecutionDialogHandler` is treated as secrets. Therefore, I add the eslint-disable-* comments. Same reason for other eslint-disable comments.